test/Elastic.Apm.AspNetFullFramework.Tests/Soap/SoapParsingTests.cs

```diff
@@ -184,7 +184,7 @@ public class SoapParsingTests
 		public void Soap12Parser_ParsesHeaderAndBody(string soap, string expectedAction)
 		{
 			var requestStream = new MemoryStream(Encoding.UTF8.GetBytes(soap));
-			var action = SoapRequest.GetSoap12ActionFromInputStream(requestStream);
+			var action = SoapRequest.GetSoap12ActionFromInputStream(null, requestStream);
```

Comment:
nit: we have `NoopLogger` (this will need a `using` at the top of the file)
```suggestion
			var action = SoapRequest.GetSoap12ActionFromInputStream(new NoopLogger(), requestStream);
```